MagnunAVF/secret-quotes
# Feat/implement authorizer

**What is the purpose of this Pull Request?**
* Implement custom authorizer.

**What was done to achieve this purpose?**
* Necessary routes was created;
* Necessary serverless infra code (serverless.yml)  was created;
* Custom authorizer  was created;
* Login route  was created;

**How to test if this really works?**
* Login an existing user (see mocked users);
* Request a  secret quote with correct headers (Authorizer=returned token, kid= returned kid).
* Verify quote with secret word.

**Who can help review this Pull Request?**
All devs

**Is closing some issue?**
No.

**TODO**
- [ ] Deploy in dev env.
